src/WebPush.php

```diff
@@ -20,12 +20,13 @@
 class WebPush
 {
     const GCM_URL = 'https://android.googleapis.com/gcm/send';
+    const FCM_URL = 'https://fcm.googleapis.com/fcm/send';
```

Comment:
What is this used for?

When you get an FCM endpoint it means it's just normal web push protocol with VAPID.